contracts/ChildChainGaugeInjectoooor.sol

```diff
@@ -272,26 +288,27 @@ contract ChildChainGaugeInjector is ConfirmedOwner, Pausable, KeeperCompatibleIn
    */
     function sweep(address token) external onlyOwner {
         uint256 balance = IERC20(token).balanceOf(address(this));
-        emit ERC20Swept(token, owner(), balance);
         SafeERC20.safeTransfer(IERC20(token), owner(), balance);
+        emit ERC20Swept(token, owner(), balance);
     }
 
 
 
     /**
      * @notice Set distributor from the injector back to the owner.
      * @notice You will have to call set_reward_distributor back to the injector FROM the current distributor if you wish to continue using the injector
-   * @param gauge The Gauge to set distributor to injector owner
-   * @param reward_token Reward token you are setting distributor for
+     * @notice be aware that the only addresses able to call set_reward_distributor are the current distributor and balancer governance authorized accounts (the LM multisig)
+   * @param gauge The Gauge to set distributor for
+   * @param reward_token Token you are setting the distributor for
    */
     function setDistributorToOwner(address gauge, address reward_token) external onlyOwner {
         IChildChainGauge gaugeContract = IChildChainGauge(gauge);
         gaugeContract.set_reward_distributor(reward_token, owner());
```

Comment:
```suggestion
         IChildChainGauge(gauge).set_reward_distributor(reward_token, msg.sender);
```

Reply:
* no need to store `gaugeContract`
* fn is `onlyOwner`, and it's cheaper to read `msg.sender` than it is to call `owner()`